### ISSUE TITLE
How detect when $('mytable').bootstrapTable() is loaded

### ISSUE THREAD

**Issue by chespinoza** [wenzhixin/bootstrap-table#536]
I'm trying to change the class .fixed-table-header properties, but I can't find where are defined,
then I'm trying to change it after bootstrapTable() is loaded, but I can't get it with on().

Thanks

**Comment by wenzhixin** [comment]
change the class? sorry i am not very clever

**Comment by chespinoza** [comment]
I want to change this generated code 
<div class="fixed-table-header" style="height: 37px; border-bottom-width: 1px; border-bottom-style: solid; border-bottom-color: rgb(221, 221, 221); margin-right: 0px;">

this prop-> border-bottom-style: solid
to -> border-bottom-style: none

How can I do that? Where is defined this code?

I could change that after the bootstrapTable was loaded with
$(".fixed-table-header").css("border-bottom-style","none");
but how can I catch it if $('mytable').bootstrapTable() seems doesn't return a loaded state?

Thanks in advance.

**Comment by NaughtyMC** [comment]
Chespinoza:

I ran into a similar problem when I was coding my project. It seems that a bit of code inside function BootstrapTable.prototype.resetHeader is called using the "setTimeout" function. This means that there is a slight pause before the headers are rendered, so any scripts done at document load are going to throw an exception if they try to alter the table headers.

What I did is add a custom event (which I called "post-headers") after this setTimeout block is complete, and then hooked my code (in your case, remove border style) up to that event listener.

Let me know if you have any more questions.
